.github/workflows/publish-to-pypi.yaml

```diff
@@ -1,5 +1,9 @@
 name: Publish python PyPI
 
+on:
+  push:
+    tags:
+      - v0.0.0
```

Comment:
This will change?

Reply:
Yes! Should have put a comment next to it. 

It's only temporary because without a trigger it is causing a lot of erroneous actions to be ran